### DISCUSSION TITLE
Ребаланс одноручного и двуручного оружия в продаже

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#357] [General]

**Comment by dosinabox** [comment]
> не хватает двуручного оружия с которым у героя был бы суммарный ущерб был 180. например 90/90 или 95/85 или 85/95.

**Comment by Kor-Angar** [comment]
по ЛОРу во второй главе городской кузнец заканчивает работу на Палов и делает оружие на продажу (рыночная площадь)

**Comment by Kor-Angar** [comment]
BAU_908_Hodges оказывает не был вооружен двуручным мечём ;-)
https://github.com/Kor-Angar/G2_NotR_MDK/blob/fb531d932179a6f3476ca44f52fc52c38cbffe38/PrjGOTHIC/Story/NPC/BAU_908_Hodges.d#L20

**Comment by Kor-Angar** [comment]
у Тендора в инстанции какой то топор. хотя в оригинале был меч паладина
https://github.com/Kor-Angar/G2_NotR_MDK/blob/fb531d932179a6f3476ca44f52fc52c38cbffe38/PrjGOTHIC/Story/NPC/PAL_260_Tandor.d#L19

**Comment by dosinabox** [comment]
> у Тендора в инстанции какой то топор. хотя в оригинале был меч паладина

потому что он всё равно нацепит топор после торговли

**Reply by Kor-Angar** [reply]
не нацепит если сделать проверки правильные на фракционное оружие. это плохо что оружие из продажи перекидывает в инстанцию. это сбивает игровую логику.

**Comment by Kor-Angar** [comment]
у Скатти одноручное оружие в оригинале 
https://github.com/Kor-Angar/G2_NotR_MDK/blob/fb531d932179a6f3476ca44f52fc52c38cbffe38/PrjGOTHIC/Story/NPC/BDT_1086_Addon_Scatty.d#L19

**Comment by Kor-Angar** [comment]
у ItMw_1H_Blessed  стоит 	flags = ITEM_SWD | ITEM_MISSION;
у ItMw_2H_Blessed  стоит 	flags = ITEM_2HD_SWD;

это значит что одноручный меч паладина можно продавать торговцам, а потом его отбирать? :-)

**Comment by dosinabox** [comment]
да

**Reply by Kor-Angar** [reply]
значит надо снимать флаг миссии с оружия паладинов

у ItMw_1H_Blessed стоит flags = ITEM_SWD | ITEM_MISSION;

**Comment by Kor-Angar** [comment]
напомни пожалуйста зачем второй топовый топор был добавлен?

**Comment by dosinabox** [comment]
> напомни пожалуйста зачем второй топовый топор был добавлен?

Во-первых, чтобы задействовать неиспользуемую текстуру.
Во-вторых, чтобы был резон его покупать. Более красивый - в продаже, менее красивый - бесплатно валяется на Ирдорате (и ещё где-то (?)).

**Reply by Kor-Angar** [reply]
Только на Ирдорате.

**Comment by Kor-Angar** [comment]
ты же так и не вернула оригинальный ассортимент товаров по оружию?

**Reply by dosinabox** [reply]
Нет. Год прошел, и я опять в отпуск) Но дело очень нужное, конечно. Хотя бы до оригинального нужно добраться.

**Reply by Kor-Angar** [reply]
отпуск это хорошо. хорошее дело отпуск. опционально можно сделать, что была возможность потом как нибудь допилить. делов тут на 10 минут, а думать про это и переписываться можно очень долго :-)

**Comment by Kor-Angar** [comment]
мне самому иногда кажется что оно должно быть вот так, а в другом прохождение уже вот эдак. и в одном случае рассуждение логичный и в другом. всему вина некачественная спайка аддона и классика. так как ЛОР(сюжет) игры подразумевает обновления ассортименты и про это есть фразы и реплики и сюжетные ходы. а вот баланс аддона не подразумевает обновления. и пираньи неубрав какие то реплики готики 2 классик начинают путать фанатов :-)

**Comment by dosinabox** [comment]
Переключатель готов. На днях пересмотрю свежим взглядом и перед отпуском выложу рабочую сборку.

**Comment by Kor-Angar** [comment]
И какой то меч на 110 урона по моему в игре не задействован был. Рунный кажется.

**Comment by dosinabox** [comment]
В тестовую сборку от 24.06.2021 добавлена опция у помощника для включения оригинального ассортимента оружия ближнего боя в продаже. Доступна только до первой торговли с любым из торговцев оружия (Эрол, Фиск, Гаретт, Хуно, Мартин, Скатти, Кантар, Хакон, Ходжес, Джора, Орлан, Ретон, Сара, Тандор).
